integration-tests/test-suite/partials/layout.xml

```diff
@@ -148,5 +148,117 @@
 			<url href="comparators/layout/success.jsp" />
 		</urls>
 	</test>
+
+	<test name="F-comparator-Layout-pixel-threshold">
```

Comment:
Please update expected number of failed/passed tests in bobcat tests (`integration-tests/sanity-functional`)